Synthetics download assertion (#6833)
DataDog/documentation: content/en/synthetics/browser_tests/actions.md

{{< img src="synthetics/browser_tests/browser_test_assertions.png" alt="Browser Test Assertion"  style="width:40%;">}}

> Assertions allow you to validate that your browser test is in the state you expect it to be in, at a given point of the simulated user journey. It is therefore particularly important to end your browser tests by an assertion to confirm your browser test ended in an expected state.

Reviewer: ```suggestion
Assertions allow you to validate that your browser test is in the state you expect it to be in, at any given point of a simulated user journey. This is why you must end your browser tests by an assertion to confirm it ended up in an expected state.
```

> Some assertions are performed on the active page. **Active page** refers to the page that was last interacted with, using a click or an assertion on a page element for instance.

Reviewer: ```suggestion
Some assertions are performed on the **active page**. Active page refers to the page that has experience the last 
 interaction like using a click or an assertion on a given page element for instance.
```

| `Test the content of the URL of the active page`          | Takes the URL of the last page that was interacted with, then asserts whether a specific value (`string`, `number`, `regex`) is present within it.                                      |
| `Test that an element is present on the active page`      | Asserts that an element (such as a specific `span`, `div`, `h`, `a`, etc.) is present on the current page.                                                                              |
| `Test that an email was received`                         | Asserts that an email was sent and whether specific values (`string`, `number`, `regex`) are present within the email subject or body. This assertion leverages [email variables][3]. |
| `Test your UI with custom JavaScript`                         | Test a custom assertion using your own JavaScript scripts. By default the assertion is done on the active page. If you want your JavaScript function to leverage a specific page element, you can select it using **Target Element** and then refer to it as the `element` parameter in your function. |

> | `Test a downloaded file`                         | Perform verifications on files downloaded in previous steps. You can assert on the file name, size and MD5 value. |

Reviewer: ```suggestion
| `Test a downloaded file`                         | Perform verifications on files downloaded in previous steps. You can assert: the file name, size, and MD5 value. |
```
